include/wx/persist/dataview.h

```diff
@@ -115,7 +117,21 @@ class wxPersistentDataViewCtrl : public wxPersistentWindow<wxDataViewCtrl>
             if ( RestoreValue(columnPrefix + wxASCII_STR(wxPERSIST_DVC_WIDTH), &width) )
                 column->SetWidth(width);
 
-            // TODO: Set the column's view position.
+            // Retrieve column's view position.
+            int pos;
+            if ( RestoreValue(columnPrefix + wxASCII_STR(wxPERSIST_DVC_POS), &pos) )
+                order.emplace(pos, column);
+        }
+
+        // Restore columns' position.
+        int new_pos = 0; // do not trust saved values from registry
```

Comment:
The values are not necessarily stored in the registry, so I'd say something like "stored values". But this comment would probably be unnecessary if we used vector instead of map anyhow.